pos_sale_order/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="account.view_account_payment_register_form" />
+        <field name="arch" type="xml">
+            <footer position="before">
+                <field name="should_display_pos_warning" invisible="1" />
+                <group
+                    attrs="{'invisible': [('should_display_pos_warning', '=', False)]}"
+                >
+                    <div>
+                        <p
+                            class="text-danger text-bf"
+                        >Warning: This payment will be done on an invoice for a point of sale order.</p>
```

Comment:
We should put something more explicit of what the user should do.

In french something like
"Attention: Cette facture provient d'une commande de point de vente. En temps normal vous ne devez pas enregistrer de paiement sur la facture mais sur la commande (sauf pour les virements bancaire).
Veuillez de préférence enregistrer le paiement sur la commande lié"

Ce paiement sera effectué sur une facture pour une commande de point de vente."